app/views/layouts/application.html.erb

```diff
@@ -3,13 +3,15 @@
   <head>
     <title><%= content_for?(:title) ? "#{yield(:title)} | " : '' %>Harvester</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@tarekraafat/autocomplete.js@10.2.7/dist/css/autoComplete.min.css">
```

Comment:
Should we prefer the packaged sources instead? `https://tarekraafat.github.io/autoComplete.js/#/installation?id=package-manager`

Reply:
Yes we should!